### DISCUSSION TITLE
Nightingale 5.15: can Feishu alert and recovery messages use different colors?

### DISCUSSION THREAD

**Discussion by evirts** [ccfos/nightingale#1402] [Q&A] [answered]
飞书机器人告警是否支持变色方案呢，如下
当有故障的时候，标题是红色
![image](https://user-images.githubusercontent.com/10336478/221231320-2ee945f2-5b62-4ca0-8444-ce325fe19080.png)
当故障恢复的时候，标题是绿色
![image](https://user-images.githubusercontent.com/10336478/221231456-12414576-d8ea-47f4-a1f1-affd5416da65.png)

**Comment by laiwei** [comment, marked as answer]
@evirts ,  请参考 @jicki 提交并合并的 PR #1388
